grails-app/conf/Config.groovy

```diff
@@ -1093,6 +1093,7 @@ openboxes {
                     menuItems: [
                             [label: "order.createPurchase.label", defaultLabel: "Create Purchase Order", href: "/${appName}/purchaseOrder/index"],
                             [label: "order.listPurchase.label", defaultLabel: "List Purchase Orders", href: "/${appName}/order/list?orderTypeCode=PURCHASE_ORDER"],
+                            [label: "location.listSuppliers.label", defaultLabel: "List Suppliers", href: "/${appName}/location/listSuppliers"],
```

Comment:
Since we have a domain class named Supplier let's change the URL to supplier/list. You can add a new SupplierController and move the listSuppliers() closure there.